### PR TITLE
Collateral symbol search: Add a possibility to search by all the words that start with a symbol

### LINKED ISSUE #524: Collateral symbol search: Add a possibility to search by all the words that start with a symbol
Related to #447, #504, #507, #523

On a positions list, search a collateral symbol returns values only when whole symbol's match is filled in into the search field. 
It would be great to add a possibility to search all the words that start with a symbol (or a combination of  symbols). For example, to show search results REN and renBTC when insert 'ren' into the search field.

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #524

### CODE REVIEW

File: src/config/networkConfig.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ const networks: { [K in NetworkIds]: Network } = {
       },
       {
         symbol: 'CDAI',
-        address: '0xa4c993e32876795abf80842adb0a241bb0eecd47',
+        address: '0x5d3a536E4D6DbD6114cc1Ead35777bAB948E3643',
```
Comment (mariano-aguero) [inline src/config/networkConfig.ts:74]: The address in mainnet was wrong

File: src/config/networkConfig.ts [hunk 2]
```diff
@@ -444,31 +444,24 @@ export class NetworkConfig {
     }
   }
 
-  getTokenFromName(tokenName: string): Maybe<Token> {
```
Comment (mariano-aguero) [inline src/config/networkConfig.ts:447]: Unnecesary method, added a new one

File: src/pages/PositionsList/index.tsx [hunk 3]
```diff
@@ -36,9 +36,8 @@ import { IconTypes } from 'components/statusInfo/common'
 import { TableControls } from 'components/table/TableControls'
 import { Hash } from 'components/text/Hash'
 import { Web3ContextStatus, useWeb3ConnectedOrInfura } from 'contexts/Web3Context'
-import { PositionWithUserBalanceWithDecimals } from 'hooks'
```
Comment (mariano-aguero) [inline src/pages/PositionsList/index.tsx:39]: Fix error

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr549--conditionaltokensexplorer.review.gnosisdev.com

**Comment by elena-zh** [comment]
I have reviewed the issue. 
Works great for me!

**Review by gabitoesmiapodo — APPROVED** [review]
giphy-downsized

**Pull request merged**